comm/src/main/protobuf/coop/rchain/comm/protocol/routing.proto

```diff
@@ -8,30 +8,35 @@ option (scalapb.options) = {
   flat_package: true
 };
 
-message Header {
-    bytes  id        = 1;
-    uint64 timestamp = 2;
-    uint64 seq       = 3;
-}
-
 message Node {
     bytes  id       = 1;
     bytes  host     = 2;
     uint32 tcp_port = 3;
     uint32 udp_port = 4;
 }
 
+message Header {
+    Node   sender         = 1;
```

Comment:
What language construct allows you to assign an integer, 1, to a class, Node?

Reply:
I'm looking to ScalaPB to answer my question.

Reply:
Haha. Yeah, it's a schema. If you serialize the schema, the integers indicate the position of that field in the serialization (more or less). This is the really low-budget way that protocol buffers maintain compatibility between versions.